salt/elife-dashboard/config/etc-nginx-sitesenabled-elife-dashboard-2.conf

```diff
@@ -0,0 +1,43 @@
+{% set app = pillar.elife_dashboard_2 %}
+
+# the upstream component nginx needs to connect to
+upstream {{ app.name }} {
+    server unix:///tmp/{{ app.name }}.sock;
+}
+
+# configuration of the server
+server {
+
+    {% if pillar.elife.env == 'dev' %}
+    listen      {{ app.web.port }};
+    server_name     localhost
+    {% else %}
+    # ssl for all non-dev environments
+    listen      {{ app.web.port }} ssl;
```

Comment:
A `listen` for non-ssl connection could be useful for local usage (e.g. `curl -v localhost/ping`). It can't be on the same port, so I usually use `app.web.port` and `app.web.port + 1`.